tests/src/test/resources/deviceregistry-jdbc/application.yml

```diff
@@ -77,3 +77,13 @@ management:
 
 server:
   port: -1
+
+---
+
+spring:
+  profiles: kafka
+hono:
+  kafka:
+    producerConfig:
```

Comment:
shouldn't this be part of *commonClientConfig*?

Reply:
It could be. But this makes only sense to me if this component had a producer and a consumer, so that one doesn't need to configure `bootstrap.servers` twice. This way it's clear that the component is a producer only.

Reply:
I see